src/main/flight/failsafe.c

```diff
@@ -138,26 +138,12 @@ static void failsafeActivate(void)
     failsafeState.phase = FAILSAFE_LANDING;
 
     ENABLE_FLIGHT_MODE(FAILSAFE_MODE);
+
     failsafeState.landingShouldBeFinishedAt = millis() + failsafeConfig()->failsafe_off_delay * MILLIS_PER_TENTH_SECOND;
 
     failsafeState.events++;
 }
 
-static void failsafeApplyControlInput(void)
-{
-#ifdef USE_GPS_RESCUE
-    if (failsafeConfig()->failsafe_procedure == FAILSAFE_PROCEDURE_GPS_RESCUE) {
-        ENABLE_FLIGHT_MODE(GPS_RESCUE_MODE);
-        return;
-    }
-#endif
-
-    for (int i = 0; i < 3; i++) {
-        rcData[i] = rxConfig()->midrc;
-    }
-    rcData[THROTTLE] = failsafeConfig()->failsafe_throttle;
```

Comment:
All channel-specific rcData changes, while in failsafe, are now performed in `detectAndApplySignalLossBehaviour`